fintrack/banks/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+
+
+class Bank(models.Model):
+    name = models.CharField(max_length=100, unique=True)
+
+    def __str__(self):
+        return self.name
+    
+
+class BankBranch(models.Model):
```

Comment:
```suggestion
class Branch(models.Model):
```

don't repeat mistakes, you can make new mistakes but no room for repeating mistakes